uiowa/uiowa
# Feature: Updated person page

Relates to #2320, although I'm not sure if it resolves it.

Adds the updated design of the person page to the default platform from Admissions.

# How to test

Mainly this is a matter of comparing the new versions of pages with the existing versions of those pages to make sure that all the same information is represented.

* `yarn workspace uids_base build`
* V3: `blt ds && drush @default.local uli people/herky-hawk`
* ^ Compare to: https://demo.sitenow.uiowa.edu/people/herky-hawk
* Admissions: `blt ds --site=admissions.uiowa.edu && drush @admissions.local uli people/amanda-cvitko-quinteiro`
* ^ Compare to: https://admissions.prod.drupal.uiowa.edu/people/amanda-cvitko-quinteiro - **Note:** These should look exactly the same, except that the background image behind the email address has been removed.
* V2: `blt ds --site=hawkeyemarchingband.uiowa.edu && drush @hawkeyemarchingband.local uli people/michael-janssen`
* ^ Compare to: https://hawkeyemarchingband.uiowa.edu/people/michael-janssen
* Person extended: `blt ds --site=informatics.grad.uiowa.edu && drush @gradinformatics.local uli research-faculty/tong-wang`
* ^ Compare to: https://informatics.grad.uiowa.edu/research-faculty/tong-wang
* Immunology: `blt ds --site=immuno.grad.uiowa.edu && drush @gradimmuno.local uli people/priya-issuree`
* ^ Compare to: https://immuno.grad.uiowa.edu/people/priya-issuree
* Visit http://gradimmuno.local.drupal.uiowa.edu/people/samuel-connell
* ^ Compare to: https://immuno.grad.uiowa.edu/people/samuel-connell
* Visit http//gradimmuno.local.drupal.uiowa.edu/people/rebecca-benson
* ^ Compare to: https://immuno.grad.uiowa.edu/people/rebecca-benson
* Check other examples you are aware of or at random!
